RRULE support in downtimes (#10738)
DataDog/documentation: content/en/monitors/downtimes.md

When a single downtime of a recurring downtime ends, the single downtime is cancelled and a new downtime is created with the same constraints and updated start and end times. **Note**: The original creator is associated to all the newly created downtimes.

{{< img src="monitors/downtimes/downtime_recurring.jpg" alt="downtime on alert"  style="width:80%;">}}

> RRULEs - or [recurrence rules][1] - are also available to define Downtimes schedule. [This official RRULE generator][2] might be helpful to generate such recurring rules.

Reviewer: ```suggestion
Use RRULEs - or [recurrence rules][1] - to define downtimes schedules. Use the official [RRULE generator][2] as a tool to generate recurring rules.
```

> Use them to define downtimes on specific days of the month. For instance, on the third Monday of each month:

Reviewer: ```suggestion
A common use case is to use RRULES to define downtimes on specific days of the month. For example, on the third Monday of each month:
```

{{< img src="monitors/downtimes/downtime_rrule.jpg" alt="downtime on alert"  style="width:80%;">}}

> **Note**: Attributes specifying the duration in RRULE are not supported (for example, `DTSTART`, `DTEND`, `DURATION`), see the RFC for the possible attributes. 

Reviewer: is there a link to the RFC?

Author: I added a link when I first introduced recurrence rules, do you think we should add another one here?

Author: But good point, we should likely remove the mention to the RFC or add a link ;) on it
